os/pm/pm_changestate.c

```diff
@@ -67,10 +67,69 @@
 
 #ifdef CONFIG_PM
 
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+#define PM_TIMER_GAP        (TIME_SLICE_TICKS * 2)
+
 /****************************************************************************
  * Private Functions
  ****************************************************************************/
 
+static void pm_timer_cb(int argc, wdparm_t arg1, ...)
+{
+	/* Do nothing here, cause we only need TIMER ISR to wake up PM,
+	 * for deceasing PM state.
+	 */
+}
+
+/****************************************************************************
+ * Name: pm_timer
+ *
+ * Description:
+ *   This internal function is called to start one timer to decrease power
+ *   state level.
+ *
+ * Input Parameters:
+ *   domain - The PM domain associated with the accumulator
+ *
+ * Returned Value:
+ *   None.
+ *
+ ****************************************************************************/
+
+static void pm_timer(int domain)
+{
+	FAR struct pm_domain_s *pdom = &g_pmglobals.domain[domain];
+	static const int pmtick[3] =
+	{
+		TIME_SLICE_TICKS * CONFIG_PM_IDLEENTER_COUNT,
+		TIME_SLICE_TICKS * CONFIG_PM_STANDBYENTER_COUNT,
+		TIME_SLICE_TICKS * CONFIG_PM_SLEEPENTER_COUNT
+	};
+
+	if (!pdom->wdog) {
+		pdom->wdog = wd_create();
+	}
+
+	if (pdom->state < PM_SLEEP && !pdom->stay[pdom->state] &&
+		pmtick[pdom->state])
+	{
```

Comment:
Let's check the coding rule.
```
if (xxx) {

}
```

Reply:
This was done since the number of characters in the line was exceeding 85 characters.

Reply:
Please check this again. 
1. Tizen coding rule says we don't follow 85 character per line.
2. Regardless of character number, you does not follow. line 122, 125, 127 and so on.

Reply:
done